orderbook/openapi.yml

```diff
@@ -240,6 +240,41 @@ paths:
                 $ref: "#/components/schemas/FeeInformation"
         404:
           description: Token non-existent or not connected to native token
+  /api/v1/price/{baseToken}/{quoteToken}:
```

Comment:
what about `/markets/{baseToken}-{quoteToken}/price`

i.e `/markets/WETH-DAI/price`

Reply:
For me, price is one property of the market. Also, makes it easier to implement different prices. 
I believe we will want to implement different prices in the future:
- spot prices
- market price
- mid market price
- some median of the last xxx period
- bid price
- ask price
- ... no idea what prices are relevant

Implementing these endpoints as properties of the market feels more natural. 
Also normally is best not to use singular form ("price"). It would be a "prices"